### ISSUE TITLE
NoSuchMethodError for RubyMine 2017.3 EAP

### ISSUE THREAD

**Issue by thestelz** [basgren/railways#29]
Error with EAP RubyMine:

```
org.jetbrains.plugins.ruby.gem.GemsRunner.runGemsExecutableScript(Lcom/intellij/openapi/projectRoots/Sdk;Lcom/intellij/openapi/module/Module;Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;Lcom/intellij/execution/ExecutionMode;[Ljava/lang/String;)Lcom/intellij/execution/process/ProcessOutput;
java.lang.NoSuchMethodError: org.jetbrains.plugins.ruby.gem.GemsRunner.runGemsExecutableScript(Lcom/intellij/openapi/projectRoots/Sdk;Lcom/intellij/openapi/module/Module;Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;Lcom/intellij/execution/ExecutionMode;[Ljava/lang/String;)Lcom/intellij/execution/process/ProcessOutput;
	at net.bitpot.railways.utils.RailwaysUtils.queryRakeRoutes(Unknown Source)
	at net.bitpot.railways.routesView.RoutesManager$UpdateRoutesTask.run(Unknown Source)
	at com.intellij.openapi.progress.impl.CoreProgressManager$TaskRunnable.run(CoreProgressManager.java:704)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:157)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:534)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:479)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:94)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:144)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl$2.run(ProgressManagerImpl.java:165)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:320)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

```
RubyMine 2017.3 EAP
Build #RM-173.3302.28, built on October 23, 2017
JRE: 1.8.0_152-release-1024-b5 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Mac OS X 10.13
```

**Comment by basgren** [comment]
Thank you! Looks like internal IDE API is changed again :) Will take a look.

**Comment by gencer** [comment]
As a new PR merged, I tested and it just solved my problem. For, who needs an immediate update, [download](https://github.com/basgren/railways/files/1453856/railways-0.8.7.tar.gz) this file and import.

`master` branch used to compile.

**Comment by basgren** [comment]
Thank you! I need some time to test it on new IDE version and then will publish new plugin version.

**Comment by basgren** [comment]
Closing the issue, as it should be fixed in v0.8.8.

**Comment by thestelz** [comment]
Just updated and confirmed working for

```
RubyMine 2017.3 EAP
Build #RM-173.3415.45, built on October 31, 2017
JRE: 1.8.0_152-release-1024-b6 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Mac OS X 10.13.1
```

Thanks!
